doc/programming_guide/performance_tips.rst

```diff
@@ -90,4 +90,44 @@ Text drawing performance
 ------------------------
 
 Arcade's :meth:`arcade.draw_text` can be quite slow. To speed things up, use
-text objects. See :ref:`drawing_text_objects`.
\ No newline at end of file
+text objects. See :ref:`drawing_text_objects`.
+
+Non-Arcade performance
+----------------------
+
+Sometimes, Arcade isn't responsible for the slowdown of your code. Instead of
+removing stuff and seeing if your game runs faster, you can use a profiler. A
+profiler is for measuring parts of your code and seeing how fast they run. This
+is useful so you can know what is taking up most of the time and causing your
+game to slow down.
+
+There are many great profilers out there, but one we recommend is called
+`cProfile`. You can make it test some code like this:
+
+.. code:: python
+    import cProfile
```

Comment:
```suggestion
.. code-block:: python

    import cProfile
```
Try this. The newlines are important in sphinx, and using the full `code-block` directive gets picked up better by PyCharm.

Reply:
This is also not python code, and needs to be formatted plain-text. I think that's 'text'? I forget.

Reply:
I think you commented on the wrong block, but your point is valid.